libs/cli/src/generators/challenge/generator.ts

```diff
@@ -73,7 +76,7 @@ export async function challengeGenerator(tree: Tree, options: Schema) {
 
   generateFiles(tree, join(__dirname, 'files', 'readme'), appDirectory, {
     tmpl: '',
-    projectName: names(options.name).name,
+    projectName: appProjectName,
```

Comment:
be careful. You need to create a new property. We still need projectName inside our file. 

look inside: `libs/cli/src/generators/challenge/files/readme/README.md__tmpl__` to understand how those properties are used

Reply:
I thought of creating a new property, but I observe that `projectName` is being used in 2 files where value of `projectName` should be updated.
`libs/cli/src/generators/challenge/files/docs/__challengeNumber__-__projectName__.md__tmpl__`
and 
`libs/cli/src/generators/challenge/files/readme/README.md__tmpl__`.

doc file is generated with `projectName`is like **`37-performance-demo.md`**

That is why I thought to go with this approach.

Let me know if I should create new property `appProjectName`,
 or rename `projectName` to  `appProjectName` and use it in
 `README.md__tmpl__`  and `__challengeNumber__-__projectName__.md__tmpl__`

Reply:
Yes that's true. But I prefer to name the new file `37-demo.md` instead of the long name with the directory prefix. 

So having two separate properties would be better. 

Thanks for your work and time.

Reply:
Done.
Thank you for the opportunity.